src/components/Books.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import BookItem from './BookItem';
+
+const Books = ({ bookList }) => (
+  <ul className="books-list-section">
+    {bookList.map((book) => (
+      <BookItem key={book.id} book={book} />
+    ))}
+  </ul>
+);
+
+Books.propTypes = {
+  bookList: PropTypes.arrayOf(PropTypes.shape(
+    {
+      title: PropTypes.string.isRequired,
+      author: PropTypes.string.isRequired,
+      id: PropTypes.string.isRequired,
+    },
+  )).isRequired,
+};
+
```

Comment:
- [ ] After you implement your changes some issues appears in the console about prop types. It seems an error with the name of the prop `title`.

![error-11](https://user-images.githubusercontent.com/79876959/145105390-a8a0327e-dd49-4009-bdcf-eeb0ad5520cd.PNG)

Reply:
Done